### ISSUE TITLE
Problema con Test Case 18 Reversals

### ISSUE THREAD

**Issue by trinisch** [RodrigoToroIcarte/IIC2113-2023-1#206]
Hola! Estoy teniendo un problema en el test case 18 de los Reversals Tests. No entiendo porque cuando The Rock intenta jugar la carta [MANEUVER] *Hip Toss*. Info: 0F/3D, Maneuver, Grapple, y el Undertaker la revierte con la carta *Rolling Takedown*. Info: 1F/#D, Reversal, ReversalGrappleSpecial, The Rock recibe 7 de daño. No debería recibir 3 de daño?

<img width="1147" alt="Screenshot 2023-10-21 at 16 21 00" src="https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/102267219/65d8066f-5052-4c18-94b3-63d5d0299a34">

**Comment by RodrigoToroIcarte** [comment]
Es porque justo antes de jugar el `Hip Toss`, `The Rock` jugó un `Jockeying for Position` y eligió que su siguiente `grapple` haga `+4D`. Por eso su `Hip Toss` hace `7D` en vez de `3D`. Luego `The Undertaker` lo revierte con un `#D` que se convierte en `7D`.

**Comment by myndr1** [comment]
Profe tengo una duda con esta misma parte del test porque The Rock revierte una carta utilizando Jockeying for position pero en el turno anterior que era de The Undertaker, luego The Rock comienza su turno, usa su habilidad e intenta jugar Hip Toss, en el enunciado dice que el efecto de Jockeying for position cuenta solo para la siguiente carta jugada dentro de ese turno, pero en este caso el efecto añadió un bonus en el turno de The Undertaker, porque afectaría la carta Hip Toss? no logro entender muy bien este caso

**Comment by RodrigoToroIcarte** [comment]
Es porque así funciona esa _temida_ carta. Como acción, agrega el bonus a la siguiente carta que juegues en tu turno. Pero como reversal, el bonus se aplica a la primera carta jugada en el siguiente turno. Usar la habilidad no rompe el bonus porque no se jugó ninguna carta :)

**Comment by myndr1** [comment]
Cierto, se me había pasado esa parte del enunciado, muchas gracias!

**Comment by trinisch** [comment]
Muchas gracias!
